.github/workflows/test-mitosheet-frontend.yml

```diff
@@ -119,16 +121,30 @@ jobs:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
         python-version: ['3.10']
-        project: ['chromium', 'firefox', 'webkit', 'Google Chrome', 'Microsoft Edge']
+        project: 
+          - test-name: 'chromium'
+            install-name: 'chromium'
+          - test-name: 'firefox'
+            install-name: 'firefox'
+          - test-name: 'webkit'
+            install-name: 'webkit'
+          - test-name: 'Google Chrome'
```

Comment:
How come we don't have Miscrosoft Edge anymore?

Reply:
Great catch